src/libopensc/card.c

```diff
@@ -397,41 +397,25 @@ int sc_lock(sc_card_t *card)
 		return r;
 	if (card->lock_count == 0) {
 		if (card->reader->ops->lock != NULL) {
-			r = card->reader->ops->lock(card->reader);
-			while (r == SC_ERROR_CARD_RESET || r == SC_ERROR_READER_REATTACHED) {
+			r = card->reader->ops->lock(card->reader, 0);
```

Comment:
@frankmorgner:
> still contains the old locking parameter

Here I would prefer to leave this as-is, because this is a very small
change (only 5 changed places, mostly one-liners) and
I am afraid that this functionality (in contrast to a change that added
a new parameter to sc_lock()) might not be easy to add back if the
PC/SC reader driver changes too much.

In fact, I see that a significant rewrite of PC/SC reader driver is waiting to be
merged (issue  #892).
Are you planning to merge that change soon?

It looks like this rewrite is going to conflict with changes from this PR so maybe
we can wait until #892 is merged, then I will rebase and we'll see how this
change fits in the rewritten driver?